integrations/extensions/docs/elasticsearch-install-and-setup/text_embedding_deploy_and_use.md

```diff
@@ -0,0 +1,189 @@
+# How to set up and use 3rd-party text embeddings for dense vector search in Elasticsearch
+This guide demonstrates how to deploy and use a text embedding model in Elasticsearch. The model will generate vector representations for text, enabling vector similarity (k-nearest neighbours) search.
+
+Please refer to [this guide](./ICD_Elasticsearch_install_and_setup.md) first to create an Elasticsearch instance and set up Kibana if you haven't already.
+
+## Install the eland library
+Run the command below to install the [eland](https://github.com/elastic/eland) library.
+```bash
+python -m pip install "eland[pytorch]"
+```
+This library allows us to pull and deploy a 3rd-party text embedding model to our Elasticsearch instance.
+
+> CAUTION: Open source and 3rd party models are not in scope of IBM or Elastic indemnity clauses. Customers must accept relevant terms and conditions to choose or bring their own models. Additionally, IBM has not assessed Elastic's supported multi-lingual models so any use of Elastic-supported models should be understood thoroughly both with respect to the terms of use for those models and the terms of use of all of the data that was used to train those models.
+
+NOTE: Please refer to the eland library [compatibility section](https://github.com/elastic/eland?tab=readme-ov-file#compatibility) to make sure you're using compatible Python and Elasticsearch versions. 
+
+## Create environment variables for ES credentials
+  ```bash
+  export ES_URL=https://<hostname:port>
+  export ES_USER=<username>
+  export ES_PASSWORD=<password>
+  export ES_CACERT=<path-to-your-cert>
+  ```  
+You can find the credentials from the service credentials of your Elasticsearch instance.
+## Pull and deploy an embedding model
+Run the command below to pull your desired model from the [Huggingface Models Hub](https://huggingface.co/models) and deploy it on your Elasticsearch instance:
```

Comment:
For some reason, I was not able to successfully run eland_import_hub_model after running python -m pip install "eland[pytorch]".  I'm getting this error: `command not found: eland_import_hub_model`

I was able to run the equivalent docker command.  But I think it ran into the issue that since I'm using the same ES ICD instance, the model was already deployed.